notifications/notification.py

```diff
@@ -26,16 +27,16 @@ def run(self):
             server.login(username, password)
             server.sendmail(username, self.recipients, self.msg.as_string())
             server.quit()
-            print('Notifications sent!')
+            logger.info('Notifications sent!')
         except SMTPAuthenticationError:
-            print('SMTPAuthenticationError')
-            print('Cannot send notification')
+            logger.error('SMTPAuthenticationError')
+            logger.error('Cannot send notification')
```

Comment:
For debugging purposes it might be useful to actually log the actual error and traceback.